x/distribution/types/expected_keepers.go

```diff
@@ -60,3 +60,7 @@ type StakingHooks interface {
 	AfterValidatorCreated(ctx context.Context, valAddr sdk.ValAddress) error // Must be called when a validator is created
 	AfterDelegationModified(ctx context.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) error
 }
+
+type DistributionHook interface {
+	BeforeFeeCollectorSend(ctx context.Context, feeCollector sdk.ModuleAccountI) error
+}
```

Comment:
i think this can just be in `types/hooks.go`, since `expected_keepers` is mainly used for other keeper interfaces

Reply:
okay will change next monday if not urgent

Reply:
Changed in 1a8bde7e2bba1c1e299eab286282dcb6afbe376c also